facia-purger/src/main/scala/com/gu/purge/facia/Lambda.scala

```diff
@@ -54,29 +46,15 @@ class Lambda extends RequestHandler[S3Event, String] {
       .header("Fastly-Soft-Purge", "1")
       .post(EmptyJsonBody)
       .build()
-    if (STAGE == "PROD") {
+
+    if (stage == "PROD") {
       val response = httpClient.newCall(request).execute()
       println(s"Sent purge request for content with ID [$contentId]. Response from Fastly API: [${response.code}] [${response.body.string}]")
```

Comment:
Is `println` the way to log with lambda btw? Is it possible to have different level of logging (ie: .info, .warn, .error)